config/config.exs

```diff
@@ -24,15 +20,31 @@ config :logger, :console,
 config :phoenix, :json_library, Jason
 
 config :turn_junebug_expressway, :turn,
+  base_url: System.get_env("TURN_BASE_URL", "https://testapp.turn.io"),
```

Comment:
These 3 changed to make a little more sense, It will require us to change the envs but we're only in QA at the moment so not a issue